src/NexxLogic.BlobConfiguration.AspNetCore/FileProvider/BlobFileProvider.cs

```diff
@@ -56,6 +47,29 @@ public IFileInfo GetFileInfo(string subpath)
         return result;
     }
 
+    public IDirectoryContents GetDirectoryContents(string subpath) {
```

Comment:
The parameter is not used. Do we need it?

Reply:
it is part of the interface of microsoft